python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -941,15 +1032,43 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             dataRefDict['fgcmFlaggedStars'] = butlerQC.get(inputRefs.fgcmFlaggedStarsInput)
             dataRefDict['fgcmFitParameters'] = butlerQC.get(inputRefs.fgcmFitParametersInput)
 
-        fgcmDatasetDict = self._fgcmFitCycle(camera, dataRefDict)
+        fgcmDatasetDict = None
+        if self.config.doMultipleCycles:
+            # Run multiple cycles at once.
+            config = copy.copy(self.config)
+            config.update(cycleNumber=0)
+            for cycle in range(self.config.multipleCyclesFinalCycleNumber + 1):
```

Comment:
Yes, the initial cycle is 0, that is done with limited parameters free.  The final cycle (as you say, n+1) is a clean-up with no parameters free.  So that makes the min field 2.  I'll add comments/docs to this effect.